src/entrypoints/new-array.d.ts

```diff
@@ -0,0 +1,18 @@
+interface ArrayConstructor {
+  new (arrayLength?: number): unknown[];
+  (arrayLength?: number): unknown[];
+
+  // not sure if the following is needed:
+  // new <T>(arrayLength: number): T[];
+  // new <T>(...items: T[]): T[];
+  // <T>(arrayLength: number): T[];
+  // <T>(...items: T[]): T[];
+  // readonly prototype: unknown[];
+
+  // following only works with `export` or `export declare` prefixed.
+  // readonly prototype: unknown[];
+}
+
+interface Array<T> {
+  readonly prototype: unknown[];
```

Comment:
Why is this needed?

Reply:
Its not, but I was caught in the `const arr = []` issue staying on any.

Could decide not to include it and only go for `new Array()`.